docs/custom-capo.md

```diff
@@ -0,0 +1,58 @@
+# Creating a custom cluster api provider ovirt and adding it to the cluster
+
+When we develop the CAPO its usually helpful to test it against a running cluster.
```

Comment:
What's CAPO? :)  I am assuming is: Cluster API Provider OVirt (CAPO) 

I would than use: When we develop the Cluster API Provider oVirt (CAPO) its ....